integration/shared/global/service_command_test.go

```diff
@@ -284,33 +284,75 @@ var _ = Describe("service command", func() {
 						})
 
-						It("displays service instance info", func() {
-							session := helpers.CF("service", serviceInstanceName)
-							Eventually(session).Should(Say(`Showing info of service %s in org %s / space %s as %s\.\.\.`, serviceInstanceName, orgName, spaceName, userName))
-							Eventually(session).Should(Say("\n\n"))
-							Eventually(session).Should(Say(`name:\s+%s`, serviceInstanceName))
-							Consistently(session).ShouldNot(Say("shared from:"))
-							Eventually(session).Should(Say(`service:\s+%s`, service))
-							Eventually(session).Should(Say(`tags:\s+database, email`))
-							Eventually(session).Should(Say(`plan:\s+%s`, servicePlan))
-							Eventually(session).Should(Say(`description:\s+fake service`))
-							Eventually(session).Should(Say(`documentation:\s+http://documentation\.url`))
-							Eventually(session).Should(Say(`dashboard:\s+http://example\.com`))
-							Eventually(session).Should(Say("\n\n"))
-							Consistently(session).ShouldNot(Say("shared with spaces:"))
-							Eventually(session).Should(Say(`Showing status of last operation from service %s\.\.\.`, serviceInstanceName))
-							Eventually(session).Should(Say("\n\n"))
-							Eventually(session).Should(Say(`status:\s+create succeeded`))
-							Eventually(session).Should(Say("message:"))
-							Eventually(session).Should(Say(`started:\s+\d{4}-[01]\d-[0-3]\dT[0-2][0-9]:[0-5]\d:[0-5]\dZ`))
-							Eventually(session).Should(Say(`updated:\s+\d{4}-[01]\d-[0-3]\dT[0-2][0-9]:[0-5]\d:[0-5]\dZ`))
-							Eventually(session).Should(Say("\n\n"))
-							Eventually(session).Should(Say("bound apps:"))
-							Eventually(session).Should(Say(`name\s+binding name\s+status\s+message`))
-							Eventually(session).Should(Say(appName1))
-							Eventually(session).Should(Say(appName2))
+						When("cc api version < 2.125.0", func() {
+							BeforeEach(func() {
+								helpers.SkipIfVersionAtLeast(ccversion.MinVersionServiceBrokerNameV2)
+							})
+
+							It("displays service instance info", func() {
+								session := helpers.CF("service", serviceInstanceName)
+								Eventually(session).Should(Say(`Showing info of service %s in org %s / space %s as %s\.\.\.`, serviceInstanceName, orgName, spaceName, userName))
+								Eventually(session).Should(Say("\n\n"))
+								Eventually(session).Should(Say(`name:\s+%s`, serviceInstanceName))
+								Consistently(session).ShouldNot(Say("shared from:"))
+								Eventually(session).Should(Say(`service:\s+%s`, service))
+								Eventually(session).Should(Say(`tags:\s+database, email`))
+								Eventually(session).Should(Say(`plan:\s+%s`, servicePlan))
+								Eventually(session).Should(Say(`description:\s+fake service`))
+								Eventually(session).Should(Say(`documentation:\s+http://documentation\.url`))
+								Eventually(session).Should(Say(`dashboard:\s+http://example\.com`))
+								Eventually(session).Should(Say(`service broker:\s+`))
```

Comment:
This line will cause line 308 to fail - the `\s+` is not restricted to the current line, so the assertion on 306 will consume the wrong newlines. By the time the test reaches 308, the session.Out buffer has been advanced too far and it will not find the correct output.